src/install/windows-shim/bun_shim_impl.zig

```diff
@@ -795,8 +795,8 @@ fn launcher(comptime mode: LauncherMode, bun_ctx: anytype) mode.RetType() {
             null,
             null,
             1, // true
-            0,
-            null,
+            w.CREATE_UNICODE_ENVIRONMENT,
```

Comment:
```suggestion
            if (is_standalone) 0 else w.CREATE_UNICODE_ENVIRONMENT,
```

Reply:
this check confused me, for some reason i thought it was above the code that i changed

https://github.com/oven-sh/bun/blob/fab96a74ea13da04459ea7f62663c4d2fd421778/src/install/windows-shim/bun_shim_impl.zig#L817